src/ui/server/controllers/insurance/business/turnover/index.test.ts

```diff
@@ -77,7 +77,7 @@ describe('controllers/insurance/business/turnover', () => {
 
   describe('pageVariables', () => {
     it('should have correct properties', () => {
-      const result = pageVariables(referenceNumber, mockCurrencies, currencyValue);
+      const result = pageVariables(referenceNumber, mockCurrencies, currencyValue!);
```

Comment:
Are we sure `currencyValue` will never be null or undefined since we are using non-null assertion operator here?

Reply:
this is in the test.  In the main file I am casting it to a String and it should not be null